recipes/deepface/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "deepface" %}
+{% set version = "0.0.75" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deepface-{{ version }}.tar.gz
+  sha256: cb1513b2c78ccbf50ae2d2f612397fe87acafc79bf25cabd4586c08d44d7187c
+
+build:
+  entry_points:
+    - deepface = deepface.DeepFace:cli
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win]
+  # https://github.com/serengil/deepface/issues/486
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - fire >=0.4.0
+    - flask >=1.1.2
+    - gdown >=3.10.1
+    - keras >=2.2.0
+    - mtcnn >=0.1.0
+    - numpy >=1.14.0
+    - opencv >=4.5.5
+    - pandas >=0.23.4
+    - pillow >=5.2.0
+    - python
+    - retina-face >=0.0.1
+    - tensorflow >=1.9.0
+    - tqdm >=4.30.0
+
+test:
+  imports:
+    - deepface
+  commands:
+    - deepface --help
+  # Upstream uses python-opencv which is a wrapper around opencv which is provided by conda-forge already.
+  # This has a slightly different version though which is why pip check fails.
+
+about:
+  home: https://github.com/serengil/deepface
+  summary: A Lightweight Face Recognition and Facial Attribute Analysis Framework (Age, Gender, Emotion, Race) for Python
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - serengil
```

Comment:
Agreed in https://github.com/serengil/deepface/issues/486#issuecomment-1137767097